eZ/Publish/API/Repository/Values/Content/Query/Aggregation/Field/AbstractFieldStatsAggregation.php

```diff
@@ -0,0 +1,28 @@
+<?php
+
+/**
+ * @copyright Copyright (C) eZ Systems AS. All rights reserved.
+ * @license For full copyright and license information view LICENSE file distributed with this source code.
+ */
+declare(strict_types=1);
+
+namespace eZ\Publish\API\Repository\Values\Content\Query\Aggregation\Field;
+
+use eZ\Publish\API\Repository\Values\Content\Query\Aggregation\AbstractStatsAggregation;
+use eZ\Publish\API\Repository\Values\Content\Query\Aggregation\FieldAggregationInterface;
+
+abstract class AbstractFieldStatsAggregation extends AbstractStatsAggregation implements FieldAggregationInterface
+{
+    use FieldAggregationTrait;
+
+    public function __construct(
+        string $name,
+        string $contentTypeIdentifier,
+        string $fieldDefinitionIdentifier)
+    {
```

Comment:
also here and in other places, for some reason fixer is not catching those dangling parenthesis.

Reply:
Fixed in 3810d17